### PR TITLE
Add handler for rauc ::Complete callback

### PR DESCRIPTION
**Pull request by OleksandrChaika**

### CODE REVIEW

File: src/Install/Installer.cpp [hunk 1]
```diff
@@ -39,30 +39,33 @@ namespace sua {
...
         int32_t  progressNotificationLimiter = 0;
 
-        while(installing) {
+        while(_installerAgent->installing()) {
             progressPercentage = _installerAgent->getInstallProgress();
-						std::this_thread::sleep_for(2000ms);
-            count++;
-            if(progressPercentage >= 100 || count >= 120) {
-                installing = false;
+            std::this_thread::sleep_for(2000ms);
```
Comment (magdalenakusz) [inline src/Install/Installer.cpp:50]: Optional: can use constant, to be able to quickly find and adjust

File: src/Install/Installer.cpp [hunk 2]
```diff
@@ -39,30 +39,33 @@ namespace sua {
...
+            tries++;
+            if(tries >= 120) {
```
Comment (magdalenakusz) [inline src/Install/Installer.cpp:52]: optional: same, constant could be introduced

**Review by magdalenakusz — APPROVED** [review]
LGTM (code level)
just 2 optional comments, not required for merging

File: src/Install/Installer.cpp (hunk 1 above)
Comment (OleksandrChaika) [reply src/Install/Installer.cpp:50]: Ok, good point, thanks.

File: src/Install/Installer.cpp (hunk 2 above)
Comment (OleksandrChaika) [reply src/Install/Installer.cpp:52]: Ok

**Pull request merged**